binder/apt.txt

```diff
@@ -1 +0,0 @@
-radare2
```

Comment:
Maybe the file can be removed altogether?

Reply:
it is being deleted? look at the "files changed" view?

Reply:
Ah yes, now I see it.